draft-ietf-wimse-workload-creds.md

```diff
@@ -0,0 +1,611 @@
+---
+title: "WIMSE Workload Credentials"
+abbrev: "WIMSE Workload Credentials"
+category: std
+
+docname: draft-ietf-wimse-workload-creds-latest
+submissiontype: IETF  # also: "independent", "editorial", "IAB", or "IRTF"
+number:
+date:
+consensus: true
+v: 3
+area: "Applications and Real-Time"
+workgroup: "Workload Identity in Multi System Environments"
+keyword:
+ - workload
+ - identity
+venue:
+  group: "Workload Identity in Multi System Environments"
+  type: "Working Group"
+  mail: "wimse@ietf.org"
+  arch: "https://mailarchive.ietf.org/arch/browse/wimse/"
+  github: "ietf-wg-wimse/draft-ietf-wimse-s2s-protocol"
+  latest: "https://ietf-wg-wimse.github.io/draft-ietf-wimse-s2s-protocol/draft-ietf-wimse-s2s-protocol.html"
+
+author:
+ -
+    fullname: "Brian Campbell"
+    organization: "Ping Identity"
+    email: bcampbell@pingidentity.com
+ -
+    fullname: "Joe Salowey"
+    organization: CyberArk
+    email: joe@salowey.net
+ -
+    fullname: "Arndt Schwenkschuster"
+    organization: Defakto Security
+    email: arndts.ietf@gmail.com
+ -
+    fullname: "Yaron Sheffer"
+    organization: Intuit
+    email: "yaronf.ietf@gmail.com"
+
```

Comment:
I was driving carpool and/or walking my dad's dog but I think we decided to add the new guy (aka @yaroslavros) to this one too?  

```suggestion
    fullname: "Yaroslav Rosomakho"
    organization: Zscaler
    email: "yrosomakho@zscaler.com"
```